### PR TITLE
Cleanups and bugs

### PR DESCRIPTION
**Pull request by tal-az**
Endpoints logic now runs in client side so it's much faster! also notifications cleanup

### CODE REVIEW

File: controllers/endpoints.js [hunk 1]
```diff
@@ -37,6 +35,16 @@ async function getSingleEndpoint(req, res) {
 }
 
 async function getEndpoints(req, res) {
+    try {
+        res.render("endpoints.ejs", {singleEndpoint: false});
```
Comment (maziland) [inline controllers/endpoints.js]: If a single endpoint now has its own EJS, we can remove the `singleEndpoint` param.
Also in line 29 when rendering `single_endpoint.ejs`

File: public/js/endpoints.js
Comment (maziland) [inline public/js/endpoints.js:1]: A few questions:
2. Why did you deleted the Checkboxes javascript?
3. Why did you removed the addNodeModal call and the `addNode` controller?

File: public/js/notifications.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            button.innerHTML = tmpSvg;
+            button.setAttribute("data-read", readBoolStr);
+            button.setAttribute("class", buttonClass);
+            document.getElementById(id).setAttribute("class", notificationClass);
+        });
+    });
+}
+
+function sendRequest(id, readBoolStr) {
+    const url = `/api/notifications/${readBoolStr}/${id}`;
```
Comment (maziland) [inline public/js/notifications.js]: i think a better uri logic is to have the id before the readBoolStr e.g `/api/notif/144/true`

File: services/fleet.js [hunk 3]
```diff
@@ -62,6 +62,41 @@ async function buildStatistics(){
     return data;
 };
 
+async function getScriptBySingleEndpoint(endpoint) {
```
Comment (maziland) [inline services/fleet.js]: `getScriptsBySingleEndpoint(endpoint)`
is less confusing - a list of scripts is returned here

**Review by maziland — CHANGES_REQUESTED** [review]

File: public/js/endpoints.js
Comment (tal-az) [reply public/js/endpoints.js:1]: By error, worked on a not up to date main.
will resolve in merge.

File: services/fleet.js (hunk 3 above)
Comment (tal-az) [reply services/fleet.js]: Done altough it's been getScript resulting in an array way before me.

Comment (maziland) [reply services/fleet.js]: Not blaming you 😅
You can also fix the rest if you'd like

Comment (tal-az) [reply services/fleet.js]: i did <3

File: public/js/endpoints.js
Comment (tal-az) [reply public/js/endpoints.js:1]: Added both from Gaia's branch manually.
cant test if works, fleet down (syndrome) :(

**Review by maziland — APPROVED** [review]

**Pull request closed without merging**